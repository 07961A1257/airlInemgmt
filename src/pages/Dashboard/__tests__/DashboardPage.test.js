import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import DashboardPage from '../DashboardPage';

describe('Dashboard Page', () => {
  const mockStore = configureStore([]);
  const dispatchMock = () => Promise.resolve({});
  const store = mockStore({
    auth: {
      users: {},
      isAuthenticated: false
    },
    apiCallsInProgress: 0,
    ancillaryLists: [],
    passengers: []
  });

  store.dispatch = jest.fn(dispatchMock);
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(
        <Provider store={store}>
          <DashboardPage />
        </Provider>,
        { context: { store } }
      );

      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});
