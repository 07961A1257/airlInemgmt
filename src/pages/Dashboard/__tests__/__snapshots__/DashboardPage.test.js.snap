// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Page render() renders the component 1`] = `<Connect(DashboardPage) />`;
